packages/help-center/src/components/utils.tsx

```diff
@@ -100,17 +100,9 @@ export const getConversationsFromSupportInteractions = (
 	supportInteractions: SupportInteraction[]
 ) => {
 	return conversations.filter( ( conversation ) => {
-		const interaction = supportInteractions.find(
+		return supportInteractions.some(
```

Comment:
👍 

We could even make the entire function definition smaller and simpler if we ditch the `return`s:

```tsx
export const getConversationsFromSupportInteractions = (
	conversations: ZendeskConversation[],
	supportInteractions: SupportInteraction[]
) =>
	conversations.filter( ( conversation ) =>
		supportInteractions.some(
			( interaction ) => interaction.uuid === conversation.metadata?.supportInteractionId
		)
	);
```